opsdroid/events.py

```diff
@@ -116,10 +119,20 @@ def __init__(
         self.created = datetime.now()
         self.event_id = event_id
         self.raw_event = raw_event
-        self.raw_parses = {}
+        self.raw_parser = raw_parser or {}
         self.responded_to = False
         self.entities = {}
 
+    @property
+    def raw_parses(self):
+        """Helper gets raw_parser."""
+        return self.raw_parser
+
+    @raw_parses.setter
+    def raw_parses(self, val):
+        """Helper sets raw_parser."""
+        self.raw_parser = val
+
```

Comment:
As you're not doing much here I wouldn't worry about these at all. We can just directly modify the `Event.raw_parser` attribute.

Reply:
On the last commit I’ve reverted these changes and moved the name back to raw_parses 😄